cmd/hil-vpn-privop/openvpn-conf.go

```diff
@@ -1,18 +1,95 @@
 package main
 
 import (
+	"fmt"
+	"os"
+	"os/exec"
 	"text/template"
 )
 
+const configDir = "/etc/openvpn/server"
+
 // Template for the open vpn config files we generate.
 var openVpnCfgTpl = template.Must(template.New("openvpn-config").Parse(`
+# This file is automatically generated by hil-vpn-privop; do not modify manually.
+
 dev tap-hil-vpn-{{ .Name }}
 secret hil-vpn-{{ .Name }}.key
+
+# The default cipher is insecure, so we explicitly set the cipher to the openvpn
+# project's recommendation. See https://community.openvpn.net/openvpn/wiki/SWEET32
+cipher AES-256-CBC
+
+lport {{ .Port }}
+
```

Comment:
Do you think it would be worth putting the vpn logs in a file rather than writing to /var/log/messages (where everything else is)? Might make debugging easier. somewhere like this `/var/log/openvpn/{hil-vpn-ID-port}.log`
 (If we use 1 file for all we would have to use `log-append`).
 (We could tell admins to use logrotate to make sure the log files don't grow too large).